.github/workflows/cd.yml

```diff
@@ -9,9 +9,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2.3.4
       - uses: guardian/actions-setup-node@main
```

Comment:
should we target `v2.4` we're touching all instances?

Reply:
As in `guardian/actions-setup-node@v2.4` instead of `@main`? I don't have any strong feeling on which is best so yeah, happy to change these.